### PR TITLE
Fixes - unwrapped bug fix + docs and citation changes

### PR DESCRIPTION
**Pull request by dswoulf**
- Fix bug in unwrapped nodes that was causing invalid codebooks
- Added mixed content/unwrapped nodes to vignette
- Added instructions to convert codebook to XML in README
- Changed citation to match common/APA author order and added date.

### CODE REVIEW

File: vignettes/rddi_intro.Rmd [hunk 1]
```diff
@@ -152,10 +180,17 @@ codebook %>%
 
 ## Export as XML
 
-Use the `as_xml()` and `write_xml` functions in `xml2` package to convert to xml and export.
+Use the `rddi::as_xml()` and `xml2::write_xml` functions to convert to xml and export. If using mixed content nodes you have to replace the html special entities with "<" and ">" before using `xml2::write_xml`.
```
Comment (psanker) [inline vignettes/rddi_intro.Rmd]: Why not do this automatically in `as_xml()`?

**Review by psanker — CHANGES_REQUESTED** [review]
Could you provide some unit tests using `testthat` (`usethis::use_testthat()`) to check that the unwrapped bug is fixed?

File: vignettes/rddi_intro.Rmd (hunk 1 above)
Comment (dswoulf) [reply vignettes/rddi_intro.Rmd]: re `rddi::as_xml()` I just wanted to highlight it is a rddi function. Not a big deal, I'll change that back. And yes, I'll create some unit tests.

**Comment by dswoulf** [comment]
Changes complete - test checking unwrapped validation in multiple spots of the codebook and vignette edit made

**Review by psanker — APPROVED** [review]
LGTM

**Pull request merged**